doc/cephfs/administration.rst

```diff
@@ -94,12 +94,13 @@ available, then the "failed" daemon will return as a standby.
 
 ::
 
-    mds deactivate <role>
+    mds deactivate [<fs_name>:]<rank>
 
-Deactivate an MDS, causing it to flush its entire journal to
-backing RADOS objects and close all open client sessions. Deactivating an MDS
-is primarily intended for bringing down a rank after reducing the number of
-active MDS (max_mds).
+Deactivating a rank causes the MDS daemon servicing this rank to flush its
+entire journal to the backing RADOS objects and close all open client
+sessions. Deactivating a rank is primarily intended to have an MDS daemon
+rejoin the cluster as a standby after reducing the number of active MDS
```

Comment:
The previous wording is correct. The actual intent is to shrink the number of active ranks. You can add that the MDS will rejoin as a standby after deactivating the rank.